drivers/pds/lib/utils.go

```diff
@@ -1024,6 +1026,128 @@ func RunTpccWorkload(dbUser string, pdsPassword string, dnsEndpoint string, dbNa
 	return flag
 }
 
+// Creates a temporary non PDS namespace of 6 letters length randomly chosen
+func CreateTempNS() (string, bool) {
+	length := 6
+	rand.Seed(time.Now().UnixNano())
+	b := make([]byte, length)
+	rand.Read(b)
+	namespace := fmt.Sprintf("%x", b)[:length]
+	ns := &corev1.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: namespace,
+		},
+	}
+	ns, err = k8sCore.CreateNamespace(ns)
+	if err != nil {
+		log.Errorf("Error while creating namespace %v", err)
```

Comment:
you can retrun error and check for nil error check rather than using bool.
Also can you reange the method to CreateNSWithGivenLen and pass length as input param.

Reply:
Done